py-polars/tests/unit/test_format.py

```diff
@@ -417,3 +417,51 @@ def test_fmt_decimal_max_scale() -> None:
 	0.14282911023321884847623576259639164703
 ]"""
     assert result == expected
+
+
+def test_simple_project_format() -> None:
```

Comment:
This is a good candidate for `@pytest.mark.parametrize`. I can show you how to do it if you're not familiar (or check some of the other tests)